db/schema.rb

```diff
@@ -304,10 +304,10 @@
     t.integer "scharge_value_check"
     t.integer "pscharge_value_check"
     t.integer "duplicate_set_id"
+    t.integer "accessible_register"
```

Comment:
lmk if this reordering is local to me for some reason  but I'm assuming not :)

Reply:
Pretty sure I've seen this one as well.